### DISCUSSION TITLE
Where can I find definition of `config-base` preset?

### DISCUSSION THREAD

**Discussion by iamkarlson** [renovatebot/renovate#32630] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

For some project we have configured renovate using `config-base`. Nobody knows why it's used and what it actually validates but it works. I couldn't find definition of this preset on the docs website, nor in the repo. is this some kind of a special preset with all defaults? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
It was renamed to `config:recommended`

**Reply by iamkarlson** [reply]
hm, but why our renovate setup hasn't broken yet? is it only in the recent versions?

**Reply by rarkins** [reply]
Renovate performs automatic migrations

**Reply by iamkarlson** [reply]
Thank you very much! Really helps!
